build_compilers/pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: build_compilers
-version: 0.0.1
+version: 0.0.2-dev
```

Comment:
Why the breaking change bump?

Reply:
uhh, I guess cause I'm bad at semver...

Should this be a +1?

Reply:
I think it's cause I normally don't see `+1-dev` since we most often use plus releases for "this is something small I'll release right away"

Reply:
I would also be fine with a 0.1.0